### PR TITLE
Yellow fix

### PR DESCRIPTION
**Pull request by catena2w**
- Fix block structure
- Add extension

### CODE REVIEW

File: papers/yellow/block.tex [hunk 1]
```diff
@@ -0,0 +1,67 @@
+\section{Ergo Block Structure}
+Ergo block consists of 4 parts:
+
+\begin{itemize}
+    \item{\em Header } - minimal amount of data, required to synchronize the chain and check PoW correctness.
+    Also contains hashes of other sections.
+    \item{\em BlockTransactions } - sequence of transactions, included in this block.
+    \item{\em ADProofs } - proofs for transactions, included into BlockTransactions.
```
Comment (kushti) [inline papers/yellow/block.tex]: "proofs for transactions included into the corresponding BlockTransactions section of a block."

File: papers/yellow/block.tex [hunk 2]
```diff
@@ -0,0 +1,67 @@
+\section{Ergo Block Structure}
+Ergo block consists of 4 parts:
+
+\begin{itemize}
+    \item{\em Header } - minimal amount of data, required to synchronize the chain and check PoW correctness.
```
Comment (kushti) [inline papers/yellow/block.tex]: "minimal amount of data required to synchronize the chain and check PoW correctness"

File: papers/yellow/block.tex [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    height &  4 & block height  \\
+    \hline
+    extensionHash & 32 & root hash of extension section  \\
+    \hline
+    equihashSolution & 128 & solution of equihash PoW puzzle  \\
+    \hline
+\end{tabular}
+
+\vspace{1em}
+Some of this fields may be calculated by node by itself if it is in correct mode:
```
Comment (kushti) [inline papers/yellow/block.tex]: this => these

Comment (kushti) [reply papers/yellow/block.tex]: correct => a certain

File: papers/yellow/block.tex [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    \item ADProofsRoot: if(status==regular AND ADState==false AND BlocksToKeep>0).
+    \item stateRoot: if(status==regular AND ADState==false AND BlocksToKeep>0).
+\end{itemize}
+
+\subsection{Extension}
+
+Extension is a key-value storage for a variety of data.
+It contains 2 parts:
+\begin{itemize}
+    \item{\em Mandatory fields } - fields, that are known to all node in the network and may be changed
```
Comment (kushti) [inline papers/yellow/block.tex]: fields known to all the nodes in the network 

(but you probably want to say "fields which values are set via consensus rules"

File: papers/yellow/block.tex [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    via soft/hard forks only. This fields have 4 bytes key and at most 64 bytes value.
+    Value length is known for all peers and this limit only important due to soft forks -
+    it is allowed to add at most 1 new key to this section per epoch (256 blocks between
+    difficulty recalculation), if key is not known to a peer it assumes that soft fork that
+    added this key was performed.
+    \item{\em Optional fields } - random data miner may add to a block. This section contains at most 8
+    elements with 32 byte key size and at most 1024 bytes value size.
```
Comment (kushti) [inline papers/yellow/block.tex]: I propose to limit optional fields to 2 key-> value pairs with 32-bytes key and 32-bytes value

**Comment by catena2w** [comment]
will be fixed in a separate PR for issue https://github.com/ergoplatform/ergo/issues/364

**Pull request closed without merging**